src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -457,4 +475,28 @@ object Task {
       else None
     }
   }
+
+  implicit class TaskStatusComparison(val task: Task) extends AnyVal {
```

Comment:
I'd put these either into a package object or a `TaskImplicits` object, not sure we have a convention for that. I guess `TaskImplicits` would be the obvious fit.